packages/shared_preferences/shared_preferences/ios/Classes/FLTSharedPreferencesPlugin.m

```diff
@@ -8,6 +8,8 @@
 
 @implementation FLTSharedPreferencesPlugin
 
+NSString *customDomain;
```

Comment:
This is a global variable, those won't be be accepted, you need to move the to be an instance variable.